core/templates/pages/exploration-player-page/layout-directives/learner-local-nav.component.html

```diff
@@ -12,7 +12,7 @@
   <li *ngIf="feedbackOptionIsShown" placement="bottom-right"
       [ngbPopover]="popContent" triggers="manual" (click)="togglePopover()"
       class="nav-item" #feedbackPopOver="ngbPopover" [autoClose]="false">
-    <a href="" class="nav-link protractor-test-exploration-feedback-popup-link"
+    <a class="nav-link protractor-test-exploration-feedback-popup-link"
```

Comment:
Why do we need an anchor tag here if we are not specifying a hyperlink? If it is just for styling purposes, could we use a more a syntactically relevant tag and add CSS rules to style it to look the same.

Reply:
Yeah!, make sense.
changed anchor tag to div tag.
Done!!